docs/source/user/glue/modvar.dot

```diff
@@ -0,0 +1,121 @@
+digraph UML_Class_diagram {
```

Comment:
do you want to keep a .png version of this here as well?  The risk is it could get out of date, but there's nothing enforcing this stays up to date either.